docs/management_api.yml

```diff
@@ -68,52 +58,6 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
-  /users/inital:
```

Comment:
@maciejmrowiec ping

@bboozzoo you're right. how should we bump this version?

intuitively, it's a major change - api is definitely broken, so we should move to `2.0.0 = v2`.

OTOH the only client here is the gui, and it's distributed with the backend...so we could get away with a minor update.

OTOH yet - maybe some users have integrated e.g. their own scripts which use the mgmt api, and we might break them. so I'd lean tworads a major update. thoughts?